src/Core/LeanCode.Components.Startup/IWebHostBuilderExtensions.cs

```diff
@@ -16,29 +14,18 @@ public static class IWebHostBuilderExtensions
     {
         public const string SystemLoggersEntryName = LeanProgram.SystemLoggersEntryName;
 
-        public const string VaultKey = "Secrets:KeyVault:VaultUrl";
-        public const string ClientIdKey = "Secrets:KeyVault:ClientId";
-        public const string ClientSecretKey = "Secrets:KeyVault:ClientSecret";
-        public const string TenantIdKey = "Secrets:KeyVault:TenantId";
         public const string MinimumLogLevelKey = "Logging:MinimumLevel";
         public const string EnableDetailedInternalLogsKey = "Logging:EnableDetailedInternalLogs";
 
-        public static IWebHostBuilder AddAppConfigurationFromAzureKeyVault(this IWebHostBuilder builder)
+        public static IWebHostBuilder ConfigureOnNonDevelopmentEnvironment(
```

Comment:
This is too generic method (does not add much value). I would keep the old one intact (just use the now-extracted helper) and introduce new one that just adds the KeyVault. But don't be generic, let it operate on KV only.

Reply:
But it neither takes away any value and using this feels fine.